tests/test_hbg_dataset.py

```diff
@@ -23,19 +20,6 @@ def ontology():
                              [('name', 'extpexpend stimulus'), ('code', 'extpexpend')]
                              ])
 
-
```

Comment:
I actually like that these were fixtures. You can use two techniques to make things better:

1. [Fixture finalization and teardown](https://docs.pytest.org/en/latest/fixture.html#fixture-finalization-executing-teardown-code). Basically os.rm() the output_filepath as the finalizer.

2. [Fixture indirect parameterization](https://stackoverflow.com/a/33879151). This way you can pass whatever files as parameters to the fixtures that you'd like.

Reply:
Thanks for the links, this is good info.
Are you suggesting using fixtures as an alternative to saving nwb into a tmp dir?

Reply:
Ok. I was not sure myself if I chose a better approach. These two functions do not have return statements thus it seemed odd to me to use fixtures that are being passed as parameters to the test functions.

Reply:
How about the following:
- Teach the fixtures to save into a temporary directory
- And let them return the full path to the NWB files

The idea behind the fixtures is that multiple test cases can easily reference them with the smallest amount of code duplication.

Reply:
Yes, this sounds like a good idea. I will change code back to using fixtures, although I am not sure how will it reduce code duplication from the current implementation. Perhaps I am missing something.

Reply:
@sgratiy You don't need to duplicate code for the second test case which uses these downloaded files. I think currently we only have one test case for each file.